packages/intellisense/src/index.ts

```diff
@@ -464,9 +394,9 @@ export function createIntellisense(
           css && '```css\n' + css + '\n```',
           cssFeatureLinks &&
             cssFeatureLinks.length &&
-            `### CSS Features\n\n${cssFeatureLinks.join('<br>')}\n`,
+            `### CSS Features\n\n${cssFeatureLinks.join('\n\n')}\n`,
```

Comment:
If `<br>` is used, the issue shown in the image is likely to occur.
They should be on two separate lines, not crammed into one.

<img width="1102" alt="image" src="https://github.com/user-attachments/assets/119b6ebf-f1ef-4384-823d-fc1af8511fde">